index.bs

```diff
@@ -2013,6 +2013,9 @@ parameter to the {{CredentialsContainer/create()}} or {{CredentialsContainer/get
     The <dfn>type</dfn> member contains the type of the [=public key credential=] the caller is referring to.
 
     The <dfn>id</dfn> member contains the [=credential ID=] of the [=public key credential=] that the caller is referring to.
+
+    The OPTIONAL <dfn>transports</dfn> member contains a hint as to how the [=client=] might communicate with the
+    [=authenticator=] this [=public key credential=] belongs to.
```

Comment:
This is nominally fine, thanks, tho I suggest:
`The OPTIONAL <dfn>transports</dfn> member contains a hint 
regarding how the [=client=] might communicate with the 
[=managing authenticator=] of the [=public key credential=] the 
caller is referring to.`

Reply:
Thanks, done!